src/lib.rs

```diff
@@ -326,6 +604,24 @@ impl Logger {
         if !self.include_level && !self.include_line_numbers && !self.include_module_path {
             self.separator = String::new();
         }
+        // The level is set based on verbosity only if the `verbosity` method has been used and
+        // _not_ overwridden a later call to the `max_level` method. If neither the `verbosity` or
+        // `max_level` method is used, then the `DEFAULT_LEVEL` is used because it is set with the
+        // `new` function. It makes more sense to calculate the level based on verbosity _after_
+        // all configuration methods have been called as opposed to during the call to the
+        // `verbosity` method. This change enables the offset feature so that the `offset` method
+        // can be used at any time during the "building" procedure before the call to `init`.
+        // Otherwise, calling the `offset` _after_ the `verbosity` method would have no effect and
+        // be difficult to communicate this limitation to users.
```

Comment:
Good comment.

Reply:
Ah, the `offset` should be `base_level` in this comment.